Bio/PDB/Dice.py

```diff
@@ -5,6 +5,21 @@
 """Code for chopping up (dicing) a structure.
 
 This module is used internally by the Bio.PDB.extract() function.
+It extracts a section of a chain from start to end and saves
+the file in PDB format
+
+Example
+--------
+Dice 2BEG.pdb, extract chain "B" from residues 18 to 20.
+>>> from Bio.PDB import Dice
+>>> from Bio.PDB import PDBParser
+>>> parser = PDBParser()
+>>> structure = parser.get_structure("scr", "PDB/2BEG.pdb")
+>>> Dice.extract(structure, "B", 18, 20, "/tmp/2BEG_diced.pdb")
```

Comment:
Using a hard coded output file does not seem ideal - but generating any files in doctests is not ideal. Perhaps just use ``Tests/test_PDB_Dice.py`` only?